packages/wallet/src/base-locked-wallet.ts

```diff
@@ -215,6 +215,7 @@ export class BaseWalletLocked extends AbstractWallet {
     transactionRequestLike: TransactionRequestLike
   ): Promise<TransactionResponse> {
     const transactionRequest = transactionRequestify(transactionRequestLike);
+    await this.provider.addMissingVariableOutputs(transactionRequest);
```

Comment:
I think this needs to be implemented on the WalletUnlocked also, as these two methods are overridden there to apply the signature.

Reply:
i realized this because the tests were failing after we rebased lol